### PR TITLE
SINGA-474 identity operator

### PR DESCRIPTION
**Pull request by ShichengChen**
Implement ONNX Operators following https://github.com/onnx/onnx/blob/master/docs/Operators.md

### CODE REVIEW

File: test/python/test_operation.py [hunk 1]
```diff
@@ -322,6 +322,27 @@ def test_LeakyRelu(self):
...
+    def test_Identity_cpu(self):
+        x = np.array([-0.9, -0.3, -0.1, 0.1, 0.5, 0.9]).reshape(3, 2).astype(np.float32)
+        y = x.copy()
+        x = tensor.from_numpy(x)
+        x.to_device(cpu_dev)
 
+        result = autograd.identity(x)
+        dx = result.creator.backward(x.data)
+
+        np.testing.assert_array_almost_equal(tensor.to_numpy(result), y, decimal=5)
```
Comment (joddiy) [inline test/python/test_operation.py]: please add the test case of backward propagation.

File: test/python/test_operation.py [hunk 2]
```diff
@@ -322,6 +322,27 @@ def test_LeakyRelu(self):
...
+    def test_Identity_gpu(self):
+        x = np.array([-0.9, -0.3, -0.1, 0.1, 0.5, 0.9]).reshape(3, 2).astype(np.float32)
+        y = x.copy()
+        x = tensor.from_numpy(x)
+        x.to_device(gpu_dev)
+
+        result = autograd.identity(x)
+        dx = result.creator.backward(x.data)
+
+        np.testing.assert_array_almost_equal(tensor.to_numpy(result), y, decimal=5)
```
Comment (joddiy) [inline test/python/test_operation.py]: please add the test case of backward propagation.

File: test/python/test_operation.py [hunk 3]
```diff
@@ -322,6 +336,37 @@ def test_LeakyRelu(self):
         np.testing.assert_array_almost_equal(tensor.to_numpy(result), XT)
         self.check_shape(dx.shape(), (3, 2))
 
+    def test_Identity_cpu(self):
+        x = np.array([-0.9, -0.3, -0.1, 0.1, 0.5, 0.9]).reshape(3, 2).astype(np.float32)
+        y = x.copy()
+        lossf=lambda x:np.sum(x)
```
Comment (joddiy) [inline test/python/test_operation.py]: why the lossf is ```np.sum```? I think for the identity, the lossf is just ```x```.

Comment (ShichengChen) [reply test/python/test_operation.py]: The derivative of np.sum is one, so I use sum as loss function. 
The eval_gradient function only accepts loss function, so I made loss function here.

Comment (joddiy) [reply test/python/test_operation.py]: Got it.  It's ok.

**Comment by joddiy** [comment]
ready for merge

**Pull request merged**